RULES.md

```diff
@@ -67,12 +67,12 @@ The [AWS Solutions Library](https://aws.amazon.com/solutions/) offers a collecti
-| AwsSolutions-ELB5  | The ELB listener is not configured for secure (HTTPs or SSL) protocols for client communication.                                                                                          | The HTTPs or SSL protocols enable secure communication by encrypting the communication between the client and the load balancer.                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| AwsSolutions-ELB2e | The CLB does not have access logs enabled.                                                                                                                                                | Access logs allow operators to to analyze traffic patterns and identify and troubleshoot security issues.                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| AwsSolutions-ELB3  | The CLB does not have connection draining enabled.                                                                                                                                        | With Connection Draining feature enabled, if an EC2 backend instance fails health checks The CLB will not send any new requests to the unhealthy instance. However, it will still allow existing (in-flight) requests to complete for the duration of the configured timeout.                                                                                                                                                                                                                                                                                          |
```

Comment:
```suggestion
| AwsSolutions-ELB3  | The CLB does not have connection draining enabled.                                                                                                                                        | With Connection Draining feature enabled, if an EC2 backend instance fails health checks the CLB will not send any new requests to the unhealthy instance. However, it will still allow existing (in-flight) requests to complete for the duration of the configured timeout.                                                                                                                                                                                                                                                                                          |
```